src/components/accessories/patientSummary/patientSummaryByType/PatientSummaryByType.tsx

```diff
@@ -10,21 +12,40 @@ import { ORDER_BY_TYPE_PAGE_SIZE } from "../consts";
 
 import { IDispatchProps, IStateProps, SummaryType, TProps } from "../types";
 
-const header = ["date"];
-const order = ["date"];
-const label = {
-  date: "Date",
-  type: "Typology",
-  result: "Result",
-  note: "Additional notes",
-};
-
 const PatientSummaryByType: FunctionComponent<TProps> = ({
   loadSummaryData,
   isLoading,
   summaryData = [],
 }) => {
   const { t } = useTranslation();
+  const header = ["date"];
+  const order = ["date"];
+  const labels = {
+    date: t("common.date"),
+    type: t("common.type"),
+    startDate: t("therapy.startDate"),
+    endDate: t("therapy.endDate"),
+    qty: t("therapy.quantity"),
+    freqInDay: t("therapy.frequencyInDay"),
+    freqInPeriod: t("therapy.frequencyInPeriod"),
+    note: t("therapy.note"),
+    medicalId: t("therapy.medical"),
+    disease: t("opd.disease1"),
+    disease2: t("opd.disease2"),
+    disease3: t("opd.disease3"),
+    pex_height: t("examination.height"),
+    pex_weight: t("examination.weight"),
+    pex_pa_max: t("examination.ap.max"),
+    pex_pa_min: t("examination.ap.min"),
+    pex_fc: t("examination.heartrate"),
+    pex_temp: t("examination.temperature"),
+    pex_sat: t("examination.saturation"),
+    pex_note: t("examination.note"),
+    exam: t("lab.exam"),
+    material: t("lab.material"),
+    result: t("lab.result"),
+  };
```

Comment:
We can use a function or even a custom hook to generate this object, given that this is the same for `PatientSummaryByType` and `PatientSummaryByDate`.

Reply:
@tsognong do you have some news about this?

Reply:
No.

Reply:
Nope just to be sure before merge this branch